lib/hanami/model/configuration.rb

```diff
@@ -225,50 +43,24 @@ def root
         Hanami.respond_to?(:root) ? Hanami.root : Pathname.pwd
```

Comment:
For me it seems sensible if hanami-model isn't been used on a Hanami project. Raising an exception in this case would be a bit baffling, IMO.